lightning-invoice/src/payment.rs

```diff
@@ -78,10 +81,15 @@
 //! #
 //! # struct FakeRouter {}
 //! # impl Router for FakeRouter {
-//! #     fn find_route<S: Score>(
-//! #         &self, payer: &PublicKey, params: &RouteParameters, payment_hash: &PaymentHash,
-//! #         first_hops: Option<&[&ChannelDetails]>, scorer: &S
-//! #     ) -> Result<Route, LightningError> { unimplemented!() }
+//! # 	fn find_route(
+//! # 		&self, payer: &PublicKey, params: &RouteParameters, payment_hash: &PaymentHash,
+//! # 		first_hops: Option<&[&ChannelDetails]>, _inflight_htlcs: InFlightHtlcs
+//! # 	) -> Result<Route, LightningError> { unimplemented!() }
```

Comment:
Let's use four spaces instead of tabs in the docs. Forget why this was the case. May just be simpler for editors.

Reply:
Addressed in c5206b3bf646b61db957ef7dd80b79538114181d